justfile

```diff
@@ -0,0 +1,80 @@
+set quiet := true
+
+import? '../sdk-codegen/justfile'
+
+# make locally installed binaries available without a longer specifier
+export PATH := "./node_modules/.bin:" + env_var('PATH')
+
+_default:
+    just --list --unsorted
+
+# this uses positional-args so that mixed quoted and unquoted arguments
+# (like filtering for a certain test) work the way we expect
+
+# ⭐ run unit tests
+[positional-arguments]
+test *args: build
+    mocha "$@"
+
+# try to compile the example TS file to make sure exports work
+types-test: build
+    tsc --build types/test
+
+# run full integration tests by installing a bunch of packages and starting servers (slow)
+integrations-test: build
+    RUN_INTEGRATION_TESTS=1 mocha test/Integration.spec.ts
+
+# run the full test suite; you probably want `test`
+ci-test: test types-test integrations-test
+
+_build mode packageType:
+    mkdir -p {{ mode }}
+    tsc -p tsconfig.{{ mode }}.json
+    echo '{"type":"{{ packageType }}"}' > {{ mode }}/package.json
+
+[private]
+build-esm: (_build "esm" "module")
+
+[private]
+build-cjs: (_build "cjs" "commonjs")
+
+# generate CJS and ESM versions of the package; mostly used as a pre-req for other steps
+build: build-esm build-cjs
+
+# ⭐ run style checks, fixing issues if possible
+lint: (lint-check "--fix")
+
+# run style checks without changing anything
+lint-check *args:
+    eslint --ext .js,.ts . {{ args }}
+
+# reinstall dependencies, if needed
+install:
+    yarn {{ if is_dependency() == "true" { "--silent" } else { "" } }}
+
+[no-exit-message]
+[private]
+prettier *args:
+    # all the project-relevant JS code
+    prettier "{src,examples,scripts,test,types}/**/*.{ts,js}" {{ args }}
```

Comment:
This is the folders eslint cared about, plus some extras (that are already formatted)